borrowing_service/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
 
-# Create your models here.
+
+class Borrowing(models.Model):
+    borrow_date = models.DateTimeField()
+    expected_return_date = models.DateTimeField()
+    actual_return_date = models.DateTimeField()
+    book_id = models.IntegerField()
+    user_id = models.IntegerField()
```

Comment:
Idk about integer field, because we already have models for book and user. But on the other hand in requirements it says to use int.